src/pages/guides/concepts/migration.md

```diff
@@ -0,0 +1,420 @@
+
+The following diagram shows how the previous API relates to the current new APIs:
+
+![V3 V4 comparison](../concepts/v3-v4.png)
+
+### Workflow API example
+
+**V3**: `Editor create API: ccEverywhere.createDesign(inputParams);`
+
+**V4**: `Editor create API: ccEverywhere.editor.create(docConfig, appConfig, exportConfig, containerConfig);`
+
+## Parameter details
+
+### Base parameters
+
+#### Base Export Config
+
+```ts
+{
+  /**
+ * Desired asset data type for images.
+ * For image output types, host can set this property to either base64 or url. The default type for images is base64.
+ * For videos, we will always send the output as a URL irrespective of this property.
+ * @default base64
+ */
+  outputType?: AssetDataType;
+}
+```
+#### Base Container Params
+
+``` ts
+{
+  /** Id of element to which iframe must be appended. By default, it is appended to the body.*/
+  parentElementId?: string;
+  /** Maximum size boundary of the iframe.*/
+  size?: PixelSize;
+  /** Minimum size boundary of the iframe.*/
+  minSize?: PixelSize;
+  /** Padding applied to the iframe in pixels.*/
+  padding?: number;
+  /** Border Radius applied to the iframe in pixels. */
+  borderRadius?: number;
+  /** Override the background color of the iframe. By default this is as per theme. */
+  backgroundColor?: string;
+  /** Show spinner while loading target app. Default is true. */
+  showLoader?: boolean;
+  /**
+ * If target app doesn't open within this time (in ms, same as of setTimeout),
+ * the error callback is invoked with error code TARGET_LOAD_TIMED_OUT.
+ */
+  loadTimeout?: number;
+}
+```
+
+### Full editor API parameters
+
+The modal params are the same for all editor-specific APIs. We have the inheritance of input and output params, as shown below. 
+
+#### Editor App Params inherit Base App params
+
+```ts
+{
+  /** Category to show by default */
+  defaultCategory?: EditorPanelView;
+  /** Search text to pass in the editor for selected panel. */
+  categorySearchText?: string;
+  /** Canvas template type */
+  templateType?: TemplateType;
+  /** Properties to configure the Editor and Modal titles */
+  editorTitle?: string;
+  publishModalTitle?: string;
+  publishErrorModalTitle?: string;
+}
+```
+
+#### Editor Export Params inherit Base Export params
+
+```ts
+{
+  exportOptions?: ExportOptionsEditor[];
+  /**
+ * Decides whether the multiple pages can be exported
+ * @default true
+ */
+  multiPage?: boolean;
+  /**
+ * Specify the list of file types that the user can download.
+ * This can be used to limit the download options as per file types for end users.
+ * This limitation is applied to both native download and custom download scenarios.
+ */
+  allowedFileTypes?: FileType[];
+  /**
+ * Value between 0 and 1 to control the quality of the image.
+ * Currently, only supported for ImageFileType.JPEG when exporting.
+ * @minimum 0
+ * @maximum 1
+ */
+  imageQuality?: number;
+}
+```
```

Comment:
```ts
{
  exportOptions?: ExportOptionsEditor[];
  /**
 * Decides whether the multiple pages can be exported
 * @default true
 */
  multiPage?: boolean;
  /**
   * Specify the list of file types that the user can download.
   * This can be used to limit the download options for end users 
   * based on file types. This limitation is applied to both native 
   * download and custom download scenarios.
   */
  allowedFileTypes?: FileType[];
  /**
 * Value between 0 and 1 to control the quality of the image.
 * Currently, only supported for ImageFileType.JPEG when exporting.
 * @minimum 0
 * @maximum 1
 */
  imageQuality?: number;
}
```